op-challenger/game/fault/types/types.go

```diff
@@ -154,14 +154,14 @@ type Claim struct {
 }
 
 // IsRoot returns true if this claim is the root claim.
-func (c *Claim) IsRoot() bool {
+func (c Claim) IsRoot() bool {
```

Comment:
nit: Making this a value receiver makes sense given that `CounteredBy` is mutable. Though none of the current receiver functions use that field. Is this more defensive programming to avoid problems later on?

Reply:
We actually use a non-pointer type for Claim nearly everywhere and it was really annoying to have to create a pointer variant just to call `IsRoot`.  But yeah it does work out nicely that because we don't use pointers often it becomes extremely hard to accidentally mutate a claim since within the challenger we never do that (only in the dispute-mon resolution code).

Reply:
You can call pointer receivers using a non-pointer type. It should work without this change.
Either is fine. Though calling a value receiver copies the value for the function call. And the `Claim` struct is fat enough that I'm slightly, and _prematurely_, worried about performance calling these functions.

Reply:
hmmm, I'd swear I made this change in response to a compiler error but it seems happy to have them as pointer receivers now so I'll switch back to that.